spec/models/alert_spec.rb

```diff
@@ -24,6 +25,8 @@
   let(:course) { create(:course) }
   let(:revision) { create(:revision) }
   let(:user) { create(:user) }
+  let(:alert) { create(:alert) }
```

Comment:
In alert factory, it was defined so(ArticlesForDeletionAlert) with type, but resolved? function doesn't override resolvable?, so I changed class name with this. 

I'm adding type here explicitly to make it more clear.